### ISSUE TITLE
Number of needed tensors to be processed

### ISSUE THREAD

**Issue by sinamajidian** [HKU-BAL/Clair#34]
Dear Clair team

It would be great if `Clair` prints the number of needed tensors to be processed, maybe like this `Processed 5000 out of 1000000 tensors`. This can help me to know how much time it needs to finish. 

```
Delay 7 seconds before starting variant calling ...
[INFO] Using 3 CPU threads
INFO:tensorflow:Restoring parameters from /Volumes/sina2tb/model
Restoring parameters from /Volumes/sina2tb/model
Calling variants ...
Processed 1000 tensors
Processed 2000 tensors
Processed 3000 tensors
Processed 4000 tensors
Processed 5000 tensors
```

Regards,
Sina

**Comment by aquaskyline** [comment]
Thanks Sina for the suggestion. As Clair supports streamlined processing, meaning that Clair will start processing the already prepared tensors before all tensors are generated, thus Clair doesn't know how many tensors beforehand.

**Comment by sinamajidian** [comment]
I see. Thanks for your kind response. 
I reach to the idea that one can check the last line of VCF file being generated and compare the position of last called variant to the length of chromosome (or the chr compared to number of all chrs in bam file) in order to have a rough sense of the needed time. 

The reason that I asked question was experiencing several hours of run although previously it takes minutes to finish.  I found my mistake that I used different reference genome version (hg19)  as the input of Clair rather than the aligned reads (hg18). 
Thanks anyway :)
